source/includes/usage-examples/code-snippets/Watch.java

```diff
@@ -0,0 +1,28 @@
+package usage.examples;
+
+import org.bson.Document;
+
+import com.mongodb.client.ChangeStreamIterable;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.changestream.FullDocument;
+
+public class Watch {
+    public static void main( String[] args ) {
+
+        // Replace the uri string with your MongoDB deployment's connection string
+        String uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+        try (MongoClient mongoClient = MongoClients.create(uri)) {
+
+            MongoDatabase database = mongoClient.getDatabase("sample_mflix");
+            MongoCollection<Document> collection = database.getCollection("movies");
+
+            ChangeStreamIterable<Document> changeStream = database.watch()
```

Comment:
Since we're seeking full document updates and we've prominently (my opinion) mentioned passing a pipeline, I wonder if we should also show how to limit to a type of operation using "operationType"in the pipeline. For examples, see: https://developer.mongodb.com/quickstart/java-change-streams